.github/workflows/bump-homebrew-formula.yml

```diff
@@ -0,0 +1,15 @@
+on:
+  push:
```

Comment:
on push or on release?

Reply:
Push on new tags that start with v.

Would that be release on new tags that start with v instead ?

This was from the upstream example on the github action but I have not tested it.